pkg/resources/data.go

```diff
@@ -87,6 +87,7 @@ type TemplateData struct {
 	inClusterPrometheusScrapingConfigsFile           string
 
 	userClusterMLAEnabled bool
+	konnectivityEnabled   bool
```

Comment:
could you keep the same name pattern everywhere? Either choose `IsKonnectivtyEnabled` or `KonnectivityEnabled`.